projects/schematics/src/migrations/5_0/constructor-deprecations/data/cart-lib-checkout.migration.ts

```diff
@@ -0,0 +1,2679 @@
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+      {
+        className: 'CommandService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'CheckoutCostCenterConnector',
+        importPath: '@spartacus/checkout/b2b/core',
+      },
+      {
+        className: 'CheckoutQueryFacade',
+        importPath: '@spartacus/checkout/base/root',
+      },
+      {
+        className: 'EventService',
+        importPath: '@spartacus/core',
+      },
+    ],
+  },
+  {
+    class: 'CheckoutEventModule',
+    importPath: '@spartacus/checkout/core',
+    deprecatedParams: [
+      {
+        className: 'CheckoutEventBuilder',
+        importPath: '@spartacus/checkout/core',
+      },
+      {
+        className: 'CheckoutEventListener',
+        importPath: '@spartacus/checkout/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'CheckoutEventBuilder',
+        importPath: '@spartacus/checkout/core',
+      },
+      {
+        className: 'CheckoutEventListener',
+        importPath: '@spartacus/checkout/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'CheckoutDeliveryAddressEventListener',
+        importPath: '@spartacus/checkout/base/root',
+      },
+      {
+        className: 'CheckoutDeliveryModeEventListener',
+        importPath: '@spartacus/checkout/base/root',
+      },
+      {
+        className: 'CheckoutPaymentEventListener',
+        importPath: '@spartacus/checkout/base/root',
+      },
+      {
+        className: 'CheckoutLegacyStoreEventListener',
+        importPath: '@spartacus/checkout/base/root',
+      },
+    ],
+  },
+  {
+    class: 'CheckoutPageMetaResolver',
+    importPath: '@spartacus/checkout/core',
+    deprecatedParams: [
+      {
+        className: 'TranslationService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'BasePageMetaResolver',
+        importPath: '@spartacus/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'CheckoutPaymentService',
+    importPath: '@spartacus/checkout/core',
+    deprecatedParams: [
+      {
+        className: 'Store',
+        importPath: '@ngrx/store',
+      },
+      {
+        className: 'Store',
+        importPath: '@ngrx/store',
+      },
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'UserIdService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'CheckoutService',
```

Comment:
I am checking `release/4.3.x`, there is no `CheckoutService` in this constructor.